sdk/eventhub/event-hubs/tests.yml

```diff
@@ -14,6 +14,7 @@ jobs:
   - template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
     parameters:
       PackageName: "@azure/event-hubs"
+      PackagePath: "sdk/eventhub/event-hubs"
```

Comment:
Why is PackagePath being added only to a few libraries and not all?

Reply:
It's added in all the tests.yml , if I have missed any tests.yml file, then let me know

Reply:
I changed the command to run the script `npm-run-command.js`  as suggested by Brandon, which reduces the dependency of an additional parameter